### PR TITLE
storybook@0.32.0 - global css styles #globalconfig

### PR DESCRIPTION
**Pull request by dandel10n**
#globalconfig:
-  Removed `f-utils` and `fozzie-colour-pallete` from dependencies

storybook@0.31.0:
- Global css styles moved from `vue.config.js` to `scss-loader.scss` and now are imported in `preview.js` to remove style duplication in every component

f-header, f-registration, f-searchbox, f-mega-modal, f-cookie-banner:
- Removed `is-visuallyHidden` and some other css style definition as should come from `fozzie`

### CODE REVIEW

**Review by 1itvinka — DISMISSED** [review]

File: packages/components/molecules/f-mega-modal/src/components/MegaModal.vue [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <template>
     <div
         ref="megaModal"
-        :class="['c-megaModal', $style['c-megaModal'], {
-            [$style['u-overlay']]: showOverlay
-        }]"
+        :class="['c-megaModal',
+                 { 'u-overlay': showOverlay },
+                 $style['c-megaModal']]"
```
Comment (DamianMullins) [inline packages/components/molecules/f-mega-modal/src/components/MegaModal.vue]: I think this was clearer before as you see the standard class and the CSS module class side-by-side? 

It was also following the style used in the Vue docs where you define your classes first followed by your condition classes.

File: packages/components/molecules/f-searchbox/src/components/formElements/FormSearchField.vue [hunk 2]
```diff
@@ -348,28 +348,4 @@ export default {
...
-    border: 0;
-    clip: rect(0 0 0 0);
-    height: 1px;
-    margin: -1px;
-    overflow: hidden;
-    padding: 0;
-    position: absolute;
-    width: 1px;
-
-    // Extends the .is-visuallyhidden class to allow the element to be focusable when navigated to via the keyboard: h5bp.com/p
```
Comment (DamianMullins) [inline packages/components/molecules/f-searchbox/src/components/formElements/FormSearchField.vue:363]: Has this been ported out to Fozzie?

Comment (dandel10n) [reply packages/components/molecules/f-searchbox/src/components/formElements/FormSearchField.vue:363]: I think it has been copied from fozzie: https://github.com/justeat/fozzie/blob/a1e5f92a0a8f3225660fbfdf64de2e2b5b6bbf7d/src/scss/trumps/_utilities.scss#L69

File: packages/components/molecules/f-mega-modal/package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@justeat/f-mega-modal",
   "description": "Fozzie Mega Modal – A Vue.js modal component",
-  "version": "0.6.0",
+  "version": "0.6.1",
```
Comment (ashleynolan) [inline packages/components/molecules/f-mega-modal/package.json]: A small one, but these aren't patch releases – they're minor code changes, so should bump the minor versions

File: packages/components/molecules/f-searchbox/src/components/formElements/FormSearchField.vue (hunk 2 above)
Comment (DamianMullins) [reply packages/components/molecules/f-searchbox/src/components/formElements/FormSearchField.vue:363]: Ah, yes, sorry I missed that.

File: packages/components/molecules/f-mega-modal/src/components/MegaModal.vue [hunk 4]
```diff
@@ -37,7 +37,7 @@
                         button-type="secondary"
                         button-size="xsmall"
                         data-test-id="close-modal"
-                        @click.native="close">
+                        @click="close">
```
Comment (DamianMullins) [inline packages/components/molecules/f-mega-modal/src/components/MegaModal.vue]: Any particular reason for this change?

Comment (dandel10n) [reply packages/components/molecules/f-mega-modal/src/components/MegaModal.vue]: Yes, from [f-button@v0.6.0 ](https://github.com/justeat/fozzie-components/blob/master/packages/components/atoms/f-button/CHANGELOG.md#v060) there is no need to add `.native` to button events

Comment (DamianMullins) [reply packages/components/molecules/f-mega-modal/src/components/MegaModal.vue]: Nice.

**Review by DamianMullins — DISMISSED** [review]

File: packages/components/molecules/f-mega-modal/src/components/MegaModal.vue (hunk 4 above)
Comment (xander-marjoram) [reply packages/components/molecules/f-mega-modal/src/components/MegaModal.vue]: @dandel10n I'm using f-button@1.1.1 and one of my tests requires `.native` still.

Comment (dandel10n) [reply packages/components/molecules/f-mega-modal/src/components/MegaModal.vue]: Yeah, I noticed this in f-registration, investigating at the moment. I think I will remove this change from this PR and add it later when I find the answer

Comment (dandel10n) [reply packages/components/molecules/f-mega-modal/src/components/MegaModal.vue]: @xander-marjoram @DamianMullins So I found that button works without `.native` but jest tests which use `trigger('click')` don't work as [trigger works only with native elements](https://vue-test-utils.vuejs.org/api/wrapper/trigger.html). The data-test-id ends up on a native button at the end but as we add it to f-button, jest doesn't recognise it as a native button. I will play around to see how to fix this issue but for now I reverted back `.native`.

**Review by DamianMullins — DISMISSED** [review]

**Pull request closed without merging**